src/Reflection/ReflectedNode.php

```diff
@@ -35,5 +47,18 @@ public function __construct(
     ) {
         preg_match('/[^:\\\\]+$/', $qualifiedName, $matches);
         $this->name = $matches[0];
+
+        $basePath = Configure::read('basePath');
+        $repo = $this->repoMap[Configure::read('config')];
+        $tag = Configure::read('tag');
+        if (str_contains($tag, 'origin/')) {
+            $tag = str_replace('origin/', '', $tag);
+        }
+
+        $repoPath = str_replace($basePath, '', $this->source->path);
+        $githubBase = 'https://github.com/cakephp/';
+        $githubBase .= sprintf('%s/tree/%s%s', $repo, $tag, $repoPath);
+        $githubBase .= sprintf('#L%s-L%s', $this->source->startLine, $this->source->endLine);
+        $this->githubLink = $githubBase;
```

Comment:
I had to split that up, otherwise the phpcs checker kept screaming about max 120 chars per line.
If there is a better way to do that I am happy to implement it 😄